### DISCUSSION TITLE
Resume Bundle x ATS Scoring

### DISCUSSION THREAD

**Discussion by avanimathur** [recodehive/resume-pitch#85] [Q&A]
[Avani Mathur Resume.pdf](https://github.com/user-attachments/files/17005268/Avani.Mathur.Resume.pdf)
(I have sent you a dm via topmate for resume review.)

I am a second year student at IGDTUW. There will be a few opportunities in 2nd yr (like UberShe, Google Generation Scholarship , DESIS etc. ) 
and next year I will be giving internship interviews. 

Please give me suggestions on how can I improve my resume. ( https://drive.google.com/file/d/18fJuglWt9EJf387_YfBY8xMzTpoqI8jw/view?usp=sharing )

Here are some of my certifications : https://drive.google.com/drive/folders/1L6s0lNx0MIhWH8XYrXRGr0LkAzA6DnNS?usp=sharing

My github and Linkedin accounts are mentioned in the resume.
Thank You.

**Comment by sanjay-kv** [comment]
[avani.pdf](https://github.com/user-attachments/files/17088164/avani.pdf)
[ATS Version of CV (20).docx](https://github.com/user-attachments/files/17088165/ATS.Version.of.CV.20.docx)


please watch the attached videos on topmate comment
